### PR TITLE
Publish the conformance test framework as a snapshot

### PR DESCRIPTION
**Pull request by netdpb**

### CODE REVIEW

**Review by cpovirk — APPROVED** [review]

File: conformance-test-framework/build.gradle [hunk 1]
```diff
@@ -18,3 +18,37 @@ dependencies {
     implementation libs.jspecify
     implementation libs.truth
 }
+
+publishing {
+    publications {
+        conformanceTestFramework(MavenPublication) {
+            pom {
+                groupId = 'org.jspecify.conformance'
+                artifactId = 'conformance-test-framework'
```
Comment (wmdietl) [inline conformance-test-framework/build.gradle:27]: This relates to our repository name discussion. To me, the repetition of 'conformance' in the group and artifact ids doesn't look nice.

**Review by wmdietl — APPROVED** [review]
Looks good, modulo naming, which we can do when/if we restructure the repository.

File: conformance-test-framework/build.gradle (hunk 1 above)
Comment (netdpb) [reply conformance-test-framework/build.gradle:27]: We can change it in the future; I want to get something out there for now.

**Comment by netdpb** [comment]
This contributes to #107.

**Pull request merged**